core/src/components/button/button.ios.scss

```diff
@@ -172,7 +179,11 @@
 }
 
 :host(.button-large) ::slotted(ion-icon[slot="icon-only"]) {
-  font-size: 1em;
+  /**
+    * The values were provided through a native iOS app.
+    * min font size: 13.5px, default font size: 20px, max font size: 40.5px
+    */
+  font-size: dynamic-font-clamp(0.85, 16px, 2.53, 1em);
```

Comment:
Why is the default size 16 when it says 20 in the comments?

Reply:
This is due to `ion-button`, the parent, having a different font size than the typical 16px.